app/javascript/mastodon/features/video/index.js

```diff
@@ -575,7 +575,7 @@ class Video extends React.PureComponent {
           dummy={!useBlurhash}
         />
 
-        {(revealed || editable) && <video
+        {(revealed || editable) && <Video
```

Comment:
This was a manual change, as otherwise the `volume` property gets flagged as undefined from the normal `<video>` element

Reply:
What is the `Video` component?

Reply:
It's what's defined in this file up on line 98

Reply:
Can I call a `Video` component from the render method of a `Video` component?

Reply:
It's called that way in a few other files, but I'm not a React person, so I can't say 100%.
```
app\javascript\mastodon\features\picture_in_picture\index.js:
  47        player = (
  48:         <Video
  49            src={src}

app\javascript\mastodon\features\status\components\detailed_status.js:
  157          media = (
  158:           <Video
  159              preview={attachment.get('preview_url')}

app\javascript\mastodon\features\ui\components\focal_point_modal.js:
  398              {media.get('type') === 'video' && (
  399:               <Video
  400                  preview={media.get('preview_url')}

app\javascript\mastodon\features\ui\components\media_modal.js:
  160          return (
  161:           <Video
  162              preview={image.get('preview_url')}

app\javascript\mastodon\features\ui\components\video_modal.js:
  39          <div className='video-modal__container'>
  40:           <Video
  41              preview={media.get('preview_url')}
```

I have started to split some of this out to separate PRs, so it might be easier to parse

Reply:
This is obviously a bug, the `Video` react component is clearly trying to use the `video` HTML tag to implement its functionality. This code as is would cause an infinite loop.

Reply:
I think there is probably a bug in either case. For now, I've pulled out a clean version of enabling this plugin to https://github.com/mastodon/mastodon/pull/22460